app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,61 @@
+class PasswordResetsController < ApplicationController
+  before_action :find_user_by_email, only: %i(edit update)
+  before_action :valid_user, only: %i(edit update)
+  before_action :check_expiration, only: %i(edit update)
+  def new; end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
```

Comment:
```suggestion
    @user = User.find_by email: params.dig(:password_reset, :email).downcase
```
em tìm hiểu về method `dig` thử xem nhé